apps/src/templates/school-association.module.scss

```diff
@@ -1,17 +1,47 @@
-@import "color.scss";
-@import "font.scss";
+@import 'color.scss';
+@import 'font.scss';
 
 .outerContainer {
-    text-align: center;
-    display: grid;
+  text-align: center;
+  display: grid;
+  padding: 10px;
+
+  h2 {
+    margin-bottom: 5px;
+  }
 }
 
 .inputContainer {
-    justify-self: center;
-    text-align: left;
-    width: 400px;
+  justify-self: center;
+  text-align: left;
+  width: 400px;
+  display: flex;
+  flex-direction: column;
+
+  & > * {
+    width: 100%;
+  }
+
+  & > input {
+    width: initial;
+    margin-top: 5px !important;
+    border-radius: 0.25rem;
+    border: 1px solid;
+    background-color: inherit;
+    text-indent: 10px;
```

Comment:
@kelbyhawn and one more q- should this also be 0.75rem?

Reply:
Yep, also good catch!